website/docs/getting-started/configuration.md

```diff
@@ -78,6 +78,54 @@ Any datastore compatible with the S3 protocol is a valid datastore.
 
 :::
 
+## Select tables from the source
+
+You have the possibility to select which tables you want in the dump.
+
+There are two options to achieve this:
+ 1. By specifying a list of tables you want (include strategy)
+ 2. By skipping a list of tables you don't want (exclude strategy)
+
+### Include strategy (`only_tables`)
+
+Add a key named `only_tables` under the `source`:
+
+```yaml
+source:
+  connection_uri: postgres://root:password@localhost:5432/root
+  only_tables: # optional - dumps only specified tables.
+    - database: public
+      table: orders
+    - database: public
+      table: customers
+```
+
+### Exclude strategy (`skip_tables`)
+
+Add a key named `skip` under the `source`:
+
+```yaml
+source:
+  connection_uri: postgres://root:password@localhost:5432/root
+  skip: # optional - exclude from the dump the specified tables.
+    - database: public
+      table: us_states
+    - database: public
+      table: order_details
+```
+
+This will exclude from the dump the tables `us_states` and `order_details`.
+
+:::warning
+
+This will exclude the table schema AND the table data.
+:::
+
+:::warning
+
+Currently only PostgreSQL and MySQL sources support this feature.
```

Comment:
Maybe I'm wrong on this

Reply:
Also, do you know if this is working only when Replibyte is in charge of the dump?

Is the `skip` and `only_tables` working when I run something like:

```bash
cat your_dump.sql | replibyte -c conf.yaml dump create -i -s postgresql
```
?